mitmproxy/command.py

```diff
@@ -37,15 +37,16 @@ class Choice(str):
     options_command = ""
 
 
+Path = typing.NewType("Path", str)
```

Comment:
Have you considered using the following?
```python
class Path(str):
    pass
```
That would enable the `issubclass` check in [line 189](https://github.com/mitmproxy/mitmproxy/pull/2669/files#diff-c0ab5de8fa401179a3c28d8a4496437cL189) to work, but I presume there's a different issue with that approach?

Reply:
I'm neutral. NewType is the "blessed way" to declare an unadorned equivalent type, on the theory that it's zero cost. Path is not exactly performance critical so either way works. I'll merge and we can adjust as we work on this.